### DISCUSSION TITLE
Welcome to rbatis Discussions!

### DISCUSSION THREAD

**Discussion by zhuxiujia** [rbatis/rbatis#68] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by knowahammer** [comment]
Hi, I am a beginner and new to Github, not sure if I can ask questions here? I have code below:

use rbatis::rbatis::Rbatis;

#[macro_use]
extern crate lazy_static;
extern crate rbatis;
...
    let rb = Rbatis::new(); 
    rb.link("sqlite://G:/rust/database").await.unwrap();
...
    let result: Vec<Struct> = rb.list("").await.unwrap();
...

Then I got error like this: 
let result: Vec<Proxy> = rb.list("").await.unwrap();
   |                                 ^^^^(rb.list()) method not found in `Rbatis`

And the same errors ocured when I used .save() etc.. , could anybody tell me why? Thank you very much.

**Reply by zhuxiujia** [reply]
you should add this code.(It is recommended to use the CLion IDE for automatic import)
```rust 
use rbatis::crud::CRUD;
```
and  Define your table structure
```rust 
use rbatis::crud::CRUD;
let result  = rb.list::<TableStruct>("").await.unwrap();
```

how to define your TableStruct see doc  website https://rbatis.github.io/rbatis.io/#/en/?id=crudenable

**Reply by knowahammer** [reply]
Thank you! You reminded me that my "CRUD" didn't work normally. You are so great.
After adding these 2 lines:

use rbatis::crud::CRUD;
use rbatis::crud_enable;

It works well now! 

我也是中国的, 我用英语写是为了让其他人也看到这个问题.
我希望能把  lazy_static = "1.4.0"  这个库以及以下导入都写到起始教程里面
use rbatis::rbatis::Rbatis;
use rbatis::crud::CRUD;
use rbatis::crud_enable;

我之前试过diessel 和 sqlx, 都倒在了入门教程的门外, 因为入门指导的各种导入和方法都不能正常使用.

你的这个库非常人性化, 特别容易理解, 感谢你们写出这么棒的库, 我们为你和你的贡献者们感到骄傲, 加油!

**Reply by zhuxiujia** [reply]
> Thank you! You reminded me that my "CRUD" didn't work normally. You are so great.
> After adding these 2 lines:
> 
> use rbatis::crud::CRUD;
> use rbatis::crud_enable;
> 
> It works well now!
> 
> 我也是中国的, 我用英语写是为了让其他人也看到这个问题.
> 我希望能把 lazy_static = "1.4.0" 这个库以及以下导入都写到起始教程里面
> use rbatis::rbatis::Rbatis;
> use rbatis::crud::CRUD;
> use rbatis::crud_enable;
> 
> 我之前试过diessel 和 sqlx, 都倒在了入门教程的门外, 因为入门指导的各种导入和方法都不能正常使用.
> 
> 你的这个库非常人性化, 特别容易理解, 感谢你们写出这么棒的库, 我们为你和你的贡献者们感到骄傲, 加油!

说的很对，简单易读，容易理解的文档很重要。我会尽可能完善文档以及描述，也希望大家能及时指出文档说的不清楚的地方哈
